GittieLabs/lovebank-frontend
# Firebase SDK Connection

Instructions for testing (also on Trello):

The app is now connected to the Firebase SDK and Analytics package

If you need access to the Firebase project, let me know and I can add you

Test for iOS (Run on Xcode):
1. Download GoogleService-Info.plist from the Firebase Console:
- Go to Settings
- Under "Your apps", select the iOS version of Love Bank click the download button
2. Move GoogleService-Info.plist to flutter_lovebank/ios/Runner
3. Open flutter_lovebank/ios/Runner.xcworkspace in Xcode
4. Hit the Run Button at the top left of the window
5. It should build with no errors and run, displaying the three-page view screen on an emulator

Test for Android (Run on Android Studio):
1. Download google-services.json from the Firebase Console
- Go to Settings
- Under "Your apps", select the Android version of Love Bank click the download button
2. Move GoogleService-Info.plist to flutter_lovebank/android/app
3. Open Android Studio
- Select "Open an existing project"
- Open flutter_lovebank/android
4. Hit the Run Button near the top center of the window
5. It should build with no errors and run, displaying the three-page view screen on an emulator
